framework/encode/vulkan_state_writer.cpp

```diff
@@ -1401,73 +1402,76 @@ void VulkanStateWriter::ProcessImageMemory(const DeviceWrapper*
             }
         }
 
-        format::InitImageCommandHeader upload_cmd;
-
-        // Packet size without the resource data.
-        upload_cmd.meta_header.block_header.size = format::GetMetaDataBlockBaseSize(upload_cmd);
-        upload_cmd.meta_header.block_header.type = format::kMetaDataBlock;
-        upload_cmd.meta_header.meta_data_id =
-            format::MakeMetaDataId(format::ApiFamilyId::ApiFamily_Vulkan, format::MetaDataType::kInitImageCommand);
-        upload_cmd.thread_id = thread_id_;
-        upload_cmd.device_id = device_wrapper->handle_id;
-        upload_cmd.image_id  = image_wrapper->handle_id;
-        upload_cmd.aspect    = snapshot_entry.aspect;
-        upload_cmd.layout    = image_wrapper->current_layout;
-
-        if (bytes != nullptr)
+        if (image_wrapper->is_swapchain == false)
```

Comment:
`if (!image_wrapper->is_swapchain)`

Reply:
Fixed. Also changed `is_swapchain` to `is_swapchain_image`